Library/Homebrew/cmd/missing.rb

```diff
@@ -6,15 +6,40 @@ module Homebrew
   def missing
     return unless HOMEBREW_CELLAR.exist?
 
-    ff = if ARGV.named.empty?
+    formulae = if ARGV.named.empty?
       Formula.installed
     else
       ARGV.resolved_formulae
     end
+    if ARGV.json == "v1"
+      missing = print_missing_json(formulae)
+    else
+      missing = print_missing(formulae)
+    end
+    Homebrew.failed = ARGV.resolved_formulae.any? && missing.any?
```

Comment:
@UniqMartin To be clear you want `missing` and `outdated` to return 0 for empty output (none missing/outdated) and 1 for non-empty output (one or more formulae missing/outdated), correct?